frontend/src/static/css/v_authorship.scss

```diff
@@ -2,6 +2,14 @@
 
 /* Authorship */
 #tab-authorship {
+
+  .file-content {
+    background-color: mui-color('github', 'background-color');
+    border: 1px solid mui-color('github', 'border-color');
+    border-radius: 0 0 4px 4px;
+    border-top: 0;
```

Comment:
instead of adding another `border-top: 0` that sets the border-top twice (once for 1px, the other for 0), how about just `border: 0 1px 1px 1px;` and `border-colour: mui-color('GitHub', 'border')?

Reply:
Makes sense!